circle.yml

```diff
@@ -0,0 +1,20 @@
+machine:
+  services:
+    - docker
+
+dependencies:
+  override:
+    - |
+      wget -q $(curl -sS -H "Authorization: token $RELEASE_TOKEN" https://api.github.com/repos/giantswarm/architect/releases/latest | grep browser_download_url | head -n 1 | cut -d '"' -f 4)
+    - chmod +x ./architect
+    - ./architect version
+
+test:
+  override:
+    - ./architect build
+
+deployment:
+  master:
+    branch: master
+    commands:
+      - ./architect deploy
```

Comment:
Yeah, we just copy and paste the circle. Deploy does the right thing (in this case, nothing - we may extend this in future)